server/services/gameLogic.js

```diff
@@ -37,38 +37,52 @@ let cards = [];
 } */
 let votes = [];
 
-/* Return the list of players, without their assigned cards */
-exports.getPlayers = () => players.map(player => ({ username: player.username, score: player.score }));
+/* Get the current word of the game */
+exports.getWord = () => currentWord;
 
-/* Return the list of cards played this round, without who played them */
-const getCards = exports.getCards = () => cards.map(card => ({ cardId: card.cardId }));
+/* Get the current status of the game */
+exports.getStatus = () => status;
+
+/* Get the current round number */
+exports.getRoundNumber = () => roundNum;
+
+/* Get the current status of the game */
+exports.getCurrentPlayer = () => currentPlayer;
 
 /* Get the list of cards for a specific player */
-exports.getCardsByUsername = (username) => players.find(player => player.username === username).cards;
+exports.getCardsByUsername = (username) => {
+    let player = players.find(player => player.username === username)
+    if (player) return player.cards.filter(card => card.played === false)
+    else return []
```

Comment:
We might run into problems later if we're hiding this error with an empty list if that makes sense?